translator/tocwconfig/sampleConfig/appsignals_and_eks_config.yaml

```diff
@@ -1175,6 +1175,264 @@ processors:
             experimental_scale: 0
             label: ""
             label_value: ""
+        - action: insert
+          aggregation_type: ""
+          experimental_match_labels:
+            generation: gen0
+          include: process.runtime.dotnet.gc.collections.count
+          match_type: ""
+          new_name: DotNetGCGen0Count
```

Comment:
The config is getting quite large...is there a way of having a wildcard for each language?

Reply:
I can give it a try, but by peeking at the code, the chance is low.

We take label from metrics and make it part of the new metric name. On the other hand, the wildcard matching and value assignment in `metricstransform` only takes the original metric name and the new metric name into consideration,

https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/ddfadc2006e53b884f1af833c0ac7cead8ed906e/processor/metricstransformprocessor/metrics_transform_processor_otlp.go#L542

```
if newName := transform.MetricIncludeFilter.expand(transform.NewName, metric.Name()); newName != "" {
  metric.SetName(newName)
} else {
  metric.SetName(transform.NewName)
}
```

We cannot configure the new_name to follow our naming strategy (with label values included).

Reply:
Tested, same conclusion.